client.go

```diff
@@ -30,10 +30,10 @@ func newClient(kubeconfig string, crScheme *runtime.Scheme) (*clientSet, error)
 	}
 
 	if kubeconfig != "" {
-		glog.V(4).Infof("Loading kube client config from path %q", kubeconfig)
+		klog.V(4).Infof("Loading kube client config from path %q", kubeconfig)
```

Comment:
let's just fmt.Println these. I'd prefer not importing another library

Reply:
done